provider/spark.go

```diff
@@ -351,9 +354,10 @@ const (
 )
 
 type DatabricksExecutor struct {
-	client  *databricks.WorkspaceClient
-	cluster string
-	config  pc.DatabricksConfig
+	client    *databricks.WorkspaceClient
+	cluster   string
+	config    pc.DatabricksConfig
+	runClient *dbClient.DatabricksClient
```

Comment:
what is the diff between client & runclient?

Reply:
A few q's
- Is the run clients primary purpose to fetch logs from dbrix?
- Is there a case where the regular client can still connect but the runClient can fail? My concern is, if the answer to above is yes (runClient purely for log fetching), if we run on dbricks and there are no failures but there is a problem with connecting to the run client our entire dbricks executor fails.